custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -188,6 +188,7 @@ class ArgumentsOptions:
     SKIP_DEPS_INSTALL = "--skip-deps-install"
     SPARSE_COLFILE = "--sparse-column-file"
     PARAMETER_FILE = "--parameter-file"
+    STRICT_VALIDATION = "--disable-strict"
```

Comment:
how about `--disable-strict-validation`. It's longer but imo more intuitive

Reply:
sure